src/main/java/teammates/logic/core/FeedbackSessionsLogic.java

```diff
@@ -1002,14 +999,23 @@ public boolean isFeedbackSessionCompletedByStudent(
             return true;
         }
         
-        
+        String feedbackSessionName = fsa.feedbackSessionName;
+        String courseId = fsa.courseId;
         List<FeedbackQuestionAttributes> allQuestions =
-                fqLogic.getFeedbackQuestionsForStudents(feedbackSessionName,
-                        courseId);
+                fqLogic.getFeedbackQuestionsForStudents(feedbackSessionName, courseId);
         // if there is no question for students, session is complete
         return allQuestions.isEmpty();
     }
 
+    public boolean isFeedbackSessionCompletedByStudent(
```

Comment:
Check where else this method is called. There may be other places where the new method can be used instead of this one.